src/dagmc/DagMC.hpp

```diff
@@ -29,8 +33,6 @@ struct DagmcVolData {
   std::string comp_name;
 };
 
-class RayTracingInterface;
```

Comment:
Is it possible to forward declare this in a namespace to avoid moving the include here?  Not sure if that works, but it would be good to avoid includes in headers if possible.

Reply:
Sure can! Good call.